docs/table/performance.inc.rst

```diff
@@ -39,3 +39,18 @@ then use ``serialize_method='data_mask'``.
 It uses the non-masked version of data and it is faster::
 
     >>> tm.write('tm.ecsv', overwrite=True, serialize_method='data_mask') # doctest: +SKIP
+
+Reading a ``.fits`` in |Table| using
```

Comment:
Should be `Read a large FITS file in ...`, and same further down `read a large FITS file`.  For small files using `memmap=True` will likely degrade performance.

Reply:
The text still refers to reading a `.fits`, that needs to be `FITS file`.